test/common/http/conn_manager_impl_test.cc

```diff
@@ -2757,7 +2757,7 @@ TEST_F(HttpConnectionManagerImplTest, FilterContinueHeadersOnlyData) {
 
   EXPECT_CALL(*decoder_filters_[0], decodeHeaders(_, false))
       .WillOnce(InvokeWithoutArgs(
-          [&]() -> FilterHeadersStatus { return FilterHeadersStatus::ContinueHeadersOnly; }));
+          [&]() -> FilterHeadersStatus { return FilterHeadersStatus::ContinueAndEndStream; }));
```

Comment:
`Return(FilterHeadersStatus::ContinueAndEndStream)` instead of `InvokeWithoutArgs` (throughout)